### PR TITLE
feat: add constants

### PR DESCRIPTION
**Pull request by marcosalves-bitg**
This PR adds constants for resp messages and status codes

### CODE REVIEW

File: src/constants/messages.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const MESSAGE_UNAUTHORIZED = 'Unauthorized'
+export const MESSAGE_INVALID_SESSION = 'Invalid Session'
+export const MESSAGE_NOTFOUND = 'Not Found'
+export const MESSAGE_FORBIDDEN = 'Forbidden'
+export const MESSAGE_USER_NOT_FOUND = 'Usuário não encontrado'
```
Comment (alanneves) [inline src/constants/messages.ts:5]: I think we should translate it all to English? What do you think?

**Review by alanneves — CHANGES_REQUESTED** [review]

**Comment by alanneves** [comment]
Should we test constants?  🤨 just a doubt

File: src/constants/messages.ts (hunk 1 above)
Comment (marcosalves-bitg) [reply src/constants/messages.ts:5]: Way better!

**Review by alanneves — APPROVED** [review]
LGTM

**Comment by marcosalves-bitg** [comment]
> Should we test constants? raised_eyebrow just a doubt

We discussed that, @Viserion77 and me, and we thoght that it would be a "misplaced effort"... cuz their are not functions, that returns something if some condition are match... they're justs... constants...

**Review by Viserion77 — APPROVED** [review]
LGTM

**Pull request merged**